docs/dev-tools/console/console.asciidoc

```diff
@@ -1,16 +1,17 @@
 [[console-kibana]]
-== Run API requests
+== Run API requests with Console
 
-Interact with the REST APIs of {es} and {kib} with *Console*. With *Console*, you can:
+**Console** lets you interact with https://www.elastic.co/guide/en/elasticsearch/reference/current/rest-apis.html[{es} APIs] and https://www.elastic.co/docs/api[{kib} APIs] from within {kib}.
 
-* Send requests and view the responses
-* View API documentation
-* Get your request history
+[role="screenshot"]
+image::dev-tools/console/images/console.png["Console", width="75%"]
+
+To go to **Console**, find **Dev Tools** in the navigation menu or use the <<kibana-navigation-search,global search bar>>.
 
-To get started, open the main menu, click *Dev Tools*, then click *Console*.
+You can also find Console directly on certain Search solution and Elasticsearch serverless project pages, where you can expand it from the footer. This Console, called **Persistent Console**, has the same capabilities and shares the same history as the Console in **Dev Tools**.
```

Comment:
cc. @serenachou 😄 

I think it's best to try to avoid naming the "embedded Console" and just say it's available in this other modality.

But the setting to disable that mode is called "persistent" so at very least have to document that as it appears in the settings

Reply:
I 100% agree with you @ElenaStoeva but after chatting with @leemthompo we thought it'd be better to align with the way it's currently explicitly named in the product (in Advanced settings it is actually referred to as **Persistent console**).

I think "embedded" and "docked" are used internally only. As for me, I would rather avoid giving it any specific name altogether, as from the user's perspective, this is just Console surfacing in different places.

Reply:
> I think "embedded" and "docked" are used internally only? As for me, I would rather avoid giving it any specific name altogether, as from the user's perspective, this is just Console surfacing in different places.

💯 We don't have to name it the "persistent Console" we can just mention the advanced setting name for how to turn it off

Reply:
> We don't have to name it the "persistent Console" we can just mention the advanced setting name for how to turn it off

Sure, this makes sense to me!

> I think "embedded" and "docked" are used internally only?

Yes, I think we call it "embedded" internally and in the codebase, but I didn't know it's actually called Persistent console in Advanced settings. 😄